File-Formats/PDF/Create-PDF-file-in-ASP-NET-Core.md

```diff
@@ -91,7 +91,7 @@ A complete work sample can be downloaded from [Create-PDF-file.zip](http://www.s
 
 
 By executing the program, you will get the PDF document as follows.
-![Generate PDF using .NET Core](GettingStarted_images/Hello_World.jpg)
+![PDF Generation output](GettingStarted_images/pdf-generation-output.jpg)
```

Comment:
add platform name in alt text as well as image name

Reply:
added in alt name, but due to common image reference, didn't changed in image name